src/tasks/FishingTask.py

```
@@ -0,0 +1,531 @@
+            if "开始钓鱼" not in name:
+                continue
+
+            x = getattr(text, "x", None)
+            y = getattr(text, "y", None)
+            width = getattr(text, "width", 0) or 0
+            height = getattr(text, "height", 0) or 0
+            if x is None or y is None:
+                continue
+
+            if mode == "raw":
+                click_x = int(round(x))
```

Ruff (0.15.11)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)